projects/1.20/assets/slashblade/slashblade/lang/zh_cn.json

```
@@ -0,0 +1,177 @@
+{
+  "itemGroup.slashblade": "拔刀剑",
+  "itemGroup.slashblade.desc": "堕入阿修罗领域",
+
+  "attribute.name.generic.reachDistance":"触及距离",
+
+  "stat.slashblade.sword_summoned": "幻影剑",
+
+  "slashblade.tooltip.killcount": "杀敌数：%1$s",
+  "slashblade.tooltip.refine": "强化等级：%1$s",
+
+  "slashblade.tooltip.material": "合成条件",
+  "slashblade.tooltip.material.requiredobjects.anvil": "合成要求场所：铁砧",
+  "slashblade.tooltip.material.killcount": "%1$s <= 杀敌数",

```

```suggestion
  "slashblade.tooltip.material.killcount": "杀敌数 不小于 %1$s",
```
或者使用`≥`、`≤`
以下也是，不过建议看看游戏内能否正常显示

---

这里显示是正常的

---

还是建议像我这么写比较好，要不然看着容易猛

---

好的